### PR TITLE
Issues/66

### PR DESCRIPTION
**Pull request by stormeye2000**

### CODE REVIEW

File: CsprSdkStandardTestsNet/Test/Utils/TestProperties.cs [hunk 1]
```diff
@@ -9,8 +9,11 @@ public class TestProperties {
         RcpPort = GetIntProperty("cspr.port.rcp", 11101);
         GetIntProperty("cspr.port.rest", 14101);
         SsePort = GetIntProperty("cspr.port.sse", 18101);
+        SpecPort = GetIntProperty("cspr.port.sse", 25101);
```
Comment (meywood) [inline CsprSdkStandardTestsNet/Test/Utils/TestProperties.cs:12]: The sse port if for events. Please fix

File: CsprSdkStandardTestsNet/Test/Steps/SpeculativeExecutionSteps.cs [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+        
+    }
+
+    private List<Transform> GetFaucetBalanceTransforms() {
+        var mainPurse = GetAccountInfo("faucet").Result.Account.MainPurse.ToString().Split("-")[1];
+        var speculativeDeployData =
+            _contextMap.Get<RpcResponse<SpeculativeExecutionResult>>(StepConstants.DEPLOY_RESULT).Parse();
+        
+        var transforms = speculativeDeployData.ExecutionResult.Effect.Transforms.Where
+            (t => t.Key.ToString().ToUpper().Equals(("balance-" + mainPurse.ToUpper()).ToUpper()));
```
Comment (meywood) [inline CsprSdkStandardTestsNet/Test/Steps/SpeculativeExecutionSteps.cs:338]: Duplicate ToUpper: mainPurse.ToUpper()).ToUpper() why not use ToLower? as ""balance-"  is lowercase

**Review by meywood — CHANGES_REQUESTED** [review]

File: CsprSdkStandardTestsNet/Test/Utils/TestProperties.cs (hunk 1 above)
Comment (stormeye2000) [reply CsprSdkStandardTestsNet/Test/Utils/TestProperties.cs:12]: done

File: CsprSdkStandardTestsNet/Test/Steps/SpeculativeExecutionSteps.cs (hunk 2 above)
Comment (stormeye2000) [reply CsprSdkStandardTestsNet/Test/Steps/SpeculativeExecutionSteps.cs:338]: done

**Pull request merged**